src/layouts/index.tsx

```diff
@@ -18,7 +18,7 @@ const Layout: React.FC<{ children: ReactNode }> = ({ children }) => {
         {children}
         <Footer />
       </Box>
-      <PriceBar />
+      {/* <PriceBar /> */}
```

Comment:
I was wondering why you need to comment <PriceBar /> out ?

Reply:
The PriceBar is the old one which will be on top of the layer and will block this popup box.

Reply:
It should be new one, please doubt check
![1](https://user-images.githubusercontent.com/51282481/169678817-7525fa22-e6e2-4d26-ba5a-efbe8dcf4c6f.png)